main/upower/APKBUILD

```diff
@@ -1,17 +1,23 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=upower
-pkgver=0.99.7
+pkgver=0.99.10
+_distver="$(echo ${pkgver} | tr '.' '_')"
```

Comment:
```suggestion
_pkgname="UPOWER_${pkgver//./_}"
```

Reply:
I support this. In general we should try avoid using bashisms so we should try avoid `${var//search/replace}`, however, i prefer that bashism over using forks in global scope, which will slow down parsing of APKBUILDs in aports tree.